packages/allure-js-commons/src/writers/FileSystemAllureWriter.ts

```diff
@@ -17,9 +17,13 @@ export class FileSystemAllureWriter implements AllureWriter {
     }
   }
 
-  writeAttachment(name: string, content: Buffer | string): void {
+  writeAttachment(name: string, content: Buffer | string, encoding?: BufferEncoding): void {
     const path = this.buildPath(name);
-    writeFileSync(path, content);
+    if (encoding) {
```

Comment:
I think, the condition is not needed here, let's set `utf-8` as default encoding and make the logic much more linear

Reply:
Updated.